invoices/invoiceregistry.go

```diff
@@ -664,21 +664,17 @@ func (i *InvoiceRegistry) cancelSingleHtlc(invoiceRef InvoiceRef,
 		// Lookup the current status of the htlc in the database.
 		var (
 			htlcState HtlcState
-			setID     *SetID
```

Comment:
Looking more closely, I think this deletion belongs to the `channeldb: split cancelHTLCs logic in the UpdateInvoice method` commit, or an independent commit.